src/components/locations/components/Locations.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Typography, Button, Row, Col, Input, Icon } from 'antd';
+const { Text, Title } = Typography;
+const Locations = () => {
+    return (
+        <div className='container-fluid'>
+            <Row >
+                <Col span={8}>
+                    <Title level={4}>
+                        Our Locations
```

Comment:
strings in constants.js

Reply:
That was Work in Progress. I paused that to fous on implementing persistant state